Despite there being a PEP-0263-compliant coding statement, pkgdepend trips over
this file because of the first line. So we delete it.

Not submitted upstream since this is a patch specific to userland and pkgdepend.


--- pip-10.0.1/src/pip/_vendor/appdirs.py.orig	2018-05-22 16:34:32.427390522 -0700
+++ pip-10.0.1/src/pip/_vendor/appdirs.py	2018-05-22 16:34:40.320413738 -0700
@@ -1,6 +1,5 @@
-#!/usr/bin/env python
 # -*- coding: utf-8 -*-
 # Copyright (c) 2005-2010 ActiveState Software Inc.
 # Copyright (c) 2013 Eddy Petri»ôor
 
 """Utilities for determining application-specific dirs.
